packages/shared/queues.ts

```diff
@@ -158,3 +158,30 @@ export const AssetPreprocessingQueue =
       keepFailedJobs: false,
     },
   );
+
+
+//Webhook worker
+export const zWebhookRequestSchema = z.object({
+  bookmarkId: z.string(),
+  url: z.string(),
```

Comment:
I don't think the `url` should be included here. We have bookmarks of other types, and you're already fetching the bookmark in the worker anyways.